xbmc/interfaces/legacy/RenderCapture.h

```diff
@@ -35,32 +34,36 @@ namespace XBMCAddon
 
     class RenderCapture : public AddonClass
     {
-      CRenderCapture* m_capture;
+      unsigned int m_captureId;
```

Comment:
sorry forgot the link:
https://github.com/bobo1on1/script.xbmc.boblight/commit/d35ae91cb26cc6042c86369a98cad70c3691c10d